paddle/fluid/framework/operator.cc

```diff
@@ -56,13 +57,22 @@ static DDim GetDims(const Scope& scope, const std::string& name,
     return DDim({-1});
   }
 
-  if (var->IsType<LoDTensor>()) {
-    return var->Get<LoDTensor>().dims();
-  } else if (var->IsType<SelectedRows>()) {
-    if (get_actual_dim) {
-      return var->Get<SelectedRows>().value().dims();
+  if (var->IsInitialized()) {
```

Comment:
worried about this will harm the performance. `GetDims` is called everywhere.

Reply:
Maybe the ```UNLIKELY``` and ```inline``` will make this better